Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -69,10 +52,18 @@ class ActivityStreamPanel: UIViewController, HomePanel {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.addSubview(tableView)
-        tableView.snp_makeConstraints { (make) in
-            make.edges.equalTo(self.view)
-        }
+
+        tableView.registerClass(SimpleHighlightCell.self, forCellReuseIdentifier: "HistoryCell")
+        tableView.registerClass(ASHorizontalScrollCell.self, forCellReuseIdentifier: "TopSiteCell")
+        tableView.backgroundColor = ASPanelUX.backgroundColor
+        tableView.separatorStyle = .None
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.separatorInset = UIEdgeInsetsZero
```

Comment:
This only exists in ios10 https://developer.apple.com/reference/uikit/uiedgeinsets/1771772-zero